app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -41,13 +41,35 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Title"
               name="title"
+              placeholder="Enter Title"
             />
             <Textarea
               required
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              placeholder="Enter Note Description"
+              rows={1}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="role"
+              options={ROLES}
+              placeholder="Select Role"
+            />
+            <Select
+              isMulti
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tags"
+              options={TAGS}
+              placeholder="Select Tag"
```

Comment:
Your label says plural `Tags`, but your placeholder says singular `Tag`. So the end user will be confused on whether they can choose single or multiple tags.